### ISSUE TITLE
The configuration screen is black

### ISSUE THREAD

**Issue by lassuly** [BloCamLimb/ModernUI#245]
I updated Modern UI.
Then when I opened the configuration screen, it turned black and couldn't see anything.
Here's the video:
https://github.com/user-attachments/assets/c9c2cc28-9a1f-4056-a31f-9df5dc57a67f
Until I update it, the configuration screen is fine.
I tried to restore the original, but I couldn't find it.

**Comment by lassuly** [comment]
This is true of Minecraft versions 1.20.4 and 1.21.1, and Modern UI is 3.11.0.4

**Comment by BloCamLimb** [comment]
I think this is duplicated with https://github.com/BloCamLimb/ModernUI/issues/240, if the Intel GPU is your only choice, then you can only wait for the next update.
